test/ApplicationTest.php

```diff
@@ -128,6 +129,20 @@ public function testPipeCanAcceptAPathArgument($middleware)
         $this->assertNull($this->app->pipe('/foo', $middleware));
     }
 
+    public function testPipeNonSlashPathOnNonStringPipeProduceTypeError()
+    {
+        // @codingStandardsIgnoreStart
+        $middleware1 = function ($request, $response) {};
+        // @codingStandardsIgnoreEnd
```

Comment:
Let's remove the CS ignore and format it in two lines. It is just 2 lines instead of 3.

Reply:
updated.